lib/tictactoe/factory/board_factory.rb

```diff
@@ -0,0 +1,24 @@
+require 'tictactoe/board'
+
+module TicTacToe
+  module Factory
+    class BoardFactory
+      TWO_BOARD = 1
```

Comment:
TWO_BOARD = 2 # seems more sensical. You don't have to start at 1 (or 0), define the constants which make sense. Looking at your "factory" method, you could just pass the constant into your constructor after doing that, maybe the factory can go away?